openapi-bal-task-plugin/src/main/java/io/ballerina/openapi/bal/tool/OpenAPICodeGeneratorTool.java

```diff
@@ -179,7 +201,8 @@ private static boolean canHandle(ToolContext toolContext) {
     /**
      * This method uses to read the openapi contract and return the {@code OpenAPI} object.
      */
-    private OpenAPI getOpenAPIContract(Path ballerinaFilePath, Path openAPIPath, Location location) {
+    private OpenAPI getOpenAPIContract(Path ballerinaFilePath, Path openAPIPath, Location location,
```

Comment:
In this method you are returning a `null` value as the result. But using `null` within a java program is not recommended. Since `Optional` is the recommended alternative to represent empty values, can't we incorporate that here ?